Modules/Core/Support/OrphanHelper.php

```
@@ -0,0 +1,50 @@
+            'DELETE FROM ip_quote_items WHERE quote_id NOT IN (SELECT quote_id FROM ip_quotes)',
+            'DELETE FROM ip_quote_item_amounts WHERE item_id NOT IN (SELECT item_id FROM ip_quote_items)',
+            'DELETE FROM ip_client_notes WHERE client_id NOT IN (SELECT client_id FROM ip_clients)',
+            'DELETE FROM ip_quote_tax_rates WHERE quote_id NOT IN (SELECT quote_id FROM ip_quotes)',
+            'DELETE FROM ip_invoice_tax_rates WHERE invoice_id NOT IN (SELECT invoice_id FROM ip_invoices)',
+            'DELETE FROM ip_invoices_recurring WHERE invoice_id NOT IN (SELECT invoice_id FROM ip_invoices)',
+        ];
+    
+        foreach ($queries as $query) {
+            $CI->db->query($query);
+        }
+    }
```

PHPMD (2.15.0)

The method delete_orphans is not named in camelCase. (undefined)

(CamelCaseMethodName)